### ISSUE TITLE
Crash when joining single-player game

### ISSUE THREAD

**Issue by eizengan** [FlashyReese/sodium-extra-fabric#285]
### Make sure you are not opening a duplicate.

- [x] [I have searched the issue tracker](https://github.com/FlashyReese/sodium-extra-fabric/issues?q=) and did not find an issue describing my crash.

### Minecraft version.

1.19.2

### Sodium Extra version.

0.4.12

### Sodium version.

0.4.4

### What happened?

Upon joining game:
- loads to 100%
- displays "Joining Game..."
- pauses briefly
- displays "Saving Game"
- CTD

The following error is displayed the next time the launcher is opened:
```
The game crashed whilst unexpected error
Error: java.lang.NullPointerException: Cannot invoke "java.lang.Integer.intValue()" because the return value of "java.util.Map.putIfAbsent(Object, Object)" is null
```

### Relevant logs

[latest.log](https://mclo.gs/gDpy4Mt)
[crash report](https://mclo.gs/S8yEdHh)

### Additional information

This is 100% reproducible on my PC with only the following installed:
- Fabric API 0.66.0
- Sodium 0.4.4
- Sodium Extra 0.4.12

Happy to dig deeper if desired, but may need a guiding hand; Java is not my home turf 😅

**Comment by gomhaille** [comment]
This appears to be the same issue as #283, which Reese just finished fixing in version `0.4.13`.
The updated jarfile with the fix is on Modrinth and CurseForge now.

**Comment by FlashyReese** [comment]
Like mentioned above, this issue has been fixed in version `0.4.13`.

**Comment by eizengan** [comment]
Nice! Sorry for the duplicate report
